front/src/modules/trainschedule/components/ManageTrainSchedule/ConstraintDistributionSwitch.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import type { Distribution } from 'common/api/osrdEditoastApi';
+import OptionsSNCF from 'common/BootstrapSNCF/OptionsSNCF';
+import { useOsrdConfActions } from 'common/osrdContext';
+import { useAppDispatch } from 'store';
+
+type Props = {
+  constraintDistribution: Distribution;
+};
+export default function ConstraintDistributionSwitch({ constraintDistribution }: Props) {
```

Comment:
On this project for components, we prefer to use `const` instead of `function`

Reply:
Done